### PR TITLE
Revamp of construction skills

### PR DESCRIPTION
**Pull request by Lamandus**
#### Summary
SUMMARY: Balance "Change around a lot of construction reciepes skills"

#### Purpose of change
Many of the skills needed don't make sense. Especially survival being one. Why do you need survival for building a fence?

#### Describe the solution
Revamping alot

#### Describe alternatives you've considered
leaving it as it is

#### Testing
works

#### Additional context
always free to tweak around some of them some more.

### CODE REVIEW

File: data/json/construction.json [hunk 1]
```diff
@@ -3442,7 +3442,6 @@
   {
     "type": "construction",
     "id": "constr_pit_shallow",
-    "skill": "survival",
     "description": "Dig a Shallow Pit",
     "category": "CONSTRUCT",
     "difficulty": 1,
```
Comment (Coolthulhu) [inline data/json/construction.json:3447]: If difficulty is non-zero, skill should be set.

File: data/json/construction.json [hunk 2]
```diff
@@ -3455,7 +3454,6 @@
   {
     "type": "construction",
     "id": "constr_pit",
-    "skill": "survival",
     "description": "Dig a Deep Pit",
     "category": "CONSTRUCT",
     "difficulty": 1,
```
Comment (Coolthulhu) [inline data/json/construction.json:3459]: Same here

File: data/json/construction.json (hunk 1 above)
Comment (Lamandus) [reply data/json/construction.json:3447]: forgot to remove difficulty, my bad!

File: data/json/construction.json (hunk 2 above)
Comment (Lamandus) [reply data/json/construction.json:3459]: forgot to remove difficulty, my bad!

**Comment by Lamandus** [comment]
it is ready to go @Coolthulhu the errors are the same as in the other PR (forge/charcoal kiln) the requirements are the same!

**Comment by Coolthulhu** [comment]
Merged
Let's hope the blueprint check system doesn't explode for no reason. It seems fragile at times.

**Pull request merged**